GenderPayGap.WebUI/Views/Scope/OutOfScopeQuestions.cshtml

```diff
@@ -99,23 +99,11 @@
                         Classes = "govuk-fieldset__legend--m"
                     },
                     Classes = "govuk-!-margin-top-6"
-                },
-                new HintViewModel
-                {
-                    Html = @<text>
-                               If you would like to read the guidance then you can find it 
-                               <a href="@Global.WhoNeedsToReportGuidanceLink" target="_blank" rel="noopener">
-                                   <span class="govuk-visually-hidden">
-                                       Find the guidance
-                                   </span>
-                                   here
-                                   <span class="govuk-visually-hidden">
-                                       (opens in a new window)
-                                   </span>
-                               </a>
-                            </text>
-                }
-                ))
+                }))
+            
+            <p class="govuk-body">
+                Read  more about <a href="@Global.WhoNeedsToReportGuidanceLink" target="_blank" rel="noopener">who needs to report their gender pay gap.</a>

```

Comment:
I noticed all the links that open a new tab have `<span class="govuk-visually-hidden">(opens in a new window)</span>` and I think we should add it here as well